genai-perf/genai_perf/export_data/exporter_config.py

```diff
@@ -40,4 +40,4 @@ class ExporterConfig:
     args: args.Namespace
     extra_inputs: Dict[str, Any]
     artifact_dir: Path
-    telemetry_stats: Optional[Dict[str, Any]] = None
```

Comment:
Telemetry_stats should never actually be None, so changing type checking to be more precise.